mne_neuron/dipole.py

```diff
@@ -15,22 +17,44 @@ def _hammfilt(x, winsz):
     win /= sum(win)
     return convolve(x, win, 'same')
 
-
-def simulate_dipole(net):
+def simulate_dipole(fn):
     """Simulate a dipole given the experiment parameters.
 
     Parameters
     ----------
-    net : Network object
-        The Network object specifying how cells are
-        connected.
+    fn : string
```

Comment:
could we call this `fname` and use str as the data type like in the rest of the codebase (hopefully this is the case -- if not it should be!)?

Reply:
dine